.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# default owners for everything in this repo
+*       @asetty @ajhfok @jfrabaute @kellyma2 @kthommandra
```

Comment:
could i get added? :)

Reply:
Oh yeah like dex.git I was thinking to add `@aristanetworks/cloudvvision-sre` (I'm guessing/hoping you're a member of this group?)

Reply:
done